src/main/java/keycontacts/model/lesson/MakeupLesson.java

```diff
@@ -0,0 +1,11 @@
+package keycontacts.model.lesson;
+
+/**
+ * Represents a makeup lesson which happens if a student misses a lesson.
+*/
+public class MakeupLesson extends RegularLesson {
+
```

Comment:
the constructor needs a javadoc (run checkstylemain before commits if you havent been)